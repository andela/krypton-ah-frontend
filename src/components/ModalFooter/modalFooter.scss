@import "../colors";
@import "../fonts";

.create-account {
  margin: 0 auto !important;

  h5 {
    font: {
      size: 0.75rem;
      weight: 200 !important;
    };

    margin: {
      bottom: 1.875rem !important;
      top: 3.125rem !important;
    }

    text-align: center;
  }

  span {
    color: $red;
    cursor: pointer;
    display: inline-block;
  }

}
