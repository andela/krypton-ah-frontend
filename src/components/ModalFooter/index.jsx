import React from 'react';
import PropTypes from 'prop-types';
import { Header } from 'semantic-ui-react';
import './modalFooter.scss';


function ModalFooter(props) {
  const { title, link } = props;
  return (
    <div className="create-account">
      <Header as="h5">
        { title }
        {' '}
        <span>{ link }</span>
      </Header>
    </div>
  );
}

ModalFooter.propTypes = {
  title: PropTypes.string,
  link: PropTypes.string
};
ModalFooter.defaultProps = {
  title: 'Create an account',
  link: 'Sign In'
};

export default ModalFooter;
