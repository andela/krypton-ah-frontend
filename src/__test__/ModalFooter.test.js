import React from 'react';
import { create } from 'react-test-renderer';
import ModalFooter from '../components/ModalFooter';

function checkFooter() {
  test('it should matches the snapshot', () => {
    const component = create(<ModalFooter />);
    expect(component.toJSON()).toMatchSnapshot();
  });
}

describe('Modal Footer', () => {
  checkFooter();
});
