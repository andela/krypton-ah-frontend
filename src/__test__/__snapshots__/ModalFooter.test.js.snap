// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal Footer it should matches the snapshot 1`] = `
<div
  className="create-account"
>
  <h5
    className="ui header"
  >
    Create an account
     
    <span>
      Sign In
    </span>
  </h5>
</div>
`;
